src/components/js/EcApplication.js

```diff
@@ -219,11 +219,11 @@ export default {
     fetchStoreApplication () {
       const { ecomApps, applicationBody } = this
       const loadPromise = applicationBody._id
-        ? ecomApps.findStoreApplication(applicationBody._id)
-        : ecomApps.fetchStoreApplications({
+        ? ecomApps.find(applicationBody._id)
+        : ecomApps.list({
           params: { app_id: this.appId }
         }).then(([app]) => {
-          return app ? ecomApps.findStoreApplication(app._id) : {}
+          return app ? this.ecomApps.find(app._id) : {}
```

Comment:
```suggestion
          return app ? ecomApps.find(app._id) : {}
```

Aqui não precisa do `this` por causa do `const { ecomApps, applicationBody } = this` :smile: